plugins/outputs/cratedb/docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+version: "3"
```

Comment:
What do we need the docker compose file for?

Reply:
I was using it for local testing. Shouldn't have checked it in. It's gone as of https://github.com/influxdata/telegraf/pull/3210/commits/a54054f906f0835731e6757b277e2e54852fe788